Added favicon size to google provider

jayenne opened the pull request:
Added favicon_size to config file

jayenne committed:
Added favicon size to google provider
Added favicon_size to config file

jayenne committed:
Add storage disk to `fetchAndSaveTo`

jayenne committed:
set default disk

stefanbauer reviewed (changes requested):
I merged a DuckDuck Provider in the meantime. I think it makes sense to add those favicon sizes to all providers as well.

jayenne committed:
Changed provider variables to camelCase
Moved config variables between provider and generator

jayenne committed:
Added new providers for Allesev, Unavatar and Yandex
unavatar is interesting because it resolves a domain, username or email to image.

jayenne committed:
Merge branch 'stefanbauer:master' into add_favicon_size_to_google_provider

stefanbauer commented on src/Provider/AllesedvProvider.php:
```diff
@@ -0,0 +1,25 @@
+    private function getUrl(string $url): string
+    {
+        $faviconSize = config('favicon-extractor.favicon_size', 32);
+        $sizeQuery = $faviconSize.'/';
```
Missing line-break before `return`

stefanbauer commented on src/Provider/GoogleProvider.php:
```diff
@@ -18,6 +18,8 @@ public function fetchFromUrl(string $url): string
     {
-        return 'https://www.google.com/s2/favicons?domain='.urlencode($url);
+        $faviconSize = config('favicon-extractor.favicon_size', 32);
+        $sizeQuery = '&sz='.$faviconSize;
```
Missing line-break before `return`

stefanbauer commented on src/Provider/YandexProvider.php:
```diff
@@ -0,0 +1,26 @@
+        // size options are limited to 16 & 32
+        $faviconSize = config('favicon-extractor.favicon_size', 32) > 16 ? 32 : 16;
+        $sizeQuery = '?size='.$faviconSize;
+        return 'https://favicon.yandex.net/favicon/'.urlencode($url).$sizeQuery;
```
Missing line-break before `return`

stefanbauer reviewed (changes requested):
I think it's good. Just those little three modifications and I'll merge it 👍🏻

stefanbauer commented:
Sorry for the delay, I missed somehow this PR.
